src/main/java/se/sundsvall/parkingpermit/integration/citizen/CitizenClient.java

```diff
@@ -0,0 +1,27 @@
+package se.sundsvall.parkingpermit.integration.citizen;
+
+import static org.springframework.http.MediaType.TEXT_PLAIN_VALUE;
+import static se.sundsvall.parkingpermit.integration.citizen.configuration.CitizenConfiguration.CLIENT_ID;
+
+import java.util.Optional;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import generated.se.sundsvall.citizen.CitizenExtended;
+import se.sundsvall.parkingpermit.integration.citizen.configuration.CitizenConfiguration;
+
+@FeignClient(name = CLIENT_ID, url = "${integration.citizen.url}", configuration = CitizenConfiguration.class)
```

Comment:
I'm not sure, but I think citizen service responds with 404 if citizen is not found (which would be the case for non residentials of Sundsvall municipality). Don't you need a `decode404 = true` here to not throw exception and instead return an optional?

Reply:
I've tested this.  (For this resource: Citizen 2.0 "GET /{personid}")

The resource used here is responding with HTTP 204 when no citizen is found.